From e3fb8fa70b702d0235ca136899bd81e30d8493d6 Mon Sep 17 00:00:00 2001
From: Kobus Goosen <kgoosen@videologyinc.com>
Date: Tue, 11 Feb 2025 15:37:52 +0100
Subject: [PATCH 4/4] buildall-isp

---
 build-all-isp.sh | 12 ++++++++++--
 1 file changed, 10 insertions(+), 2 deletions(-)

diff --git a/build-all-isp.sh b/build-all-isp.sh
index 312cd13..03b2fc3 100755
--- a/build-all-isp.sh
+++ b/build-all-isp.sh
@@ -54,7 +54,7 @@ echo "Full build --------------------->"
 	-DIMX_G2D=ON -DSDKTARGETSYSROOT=$SDKTARGETSYSROOT -DCMAKE_INSTALL_LIBDIR=$CMAKE_INSTALL_LIBDIR \
 	-DCMAKE_INSTALL_PREFIX=$CMAKE_INSTALL_PREFIX -DCMAKE_INSTALL_INCLUDEDIR=$CMAKE_INSTALL_INCLUDEDIR \
 	-Werror=date-time -Wno-dev ..
-else 
+else
 echo "Partial build --------------------->"
 
     echo "V4L2 mode build --------------------->"
@@ -100,6 +100,14 @@ cp -a appshell/build/generated/${BUILD_DIR}/bin/*.cfg ${ISP_BIN_DIR}/
 cp -a appshell/build/generated/${BUILD_DIR}/bin/*.cfg ${ISP_BIN_DIR}/
 cp -a appshell/build/generated/${BUILD_TYPE}/bin/tuningext ${ISP_BIN_DIR}
 cp -a appshell/build/generated/${BUILD_TYPE}/bin/vvext ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/IMX662* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/imx662* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/IMX676* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/imx676* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/IMX678* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/imx678* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/IMX900* ${ISP_BIN_DIR}
+cp -a appshell/build/generated/${BUILD_TYPE}/bin/imx900* ${ISP_BIN_DIR}
 
 #cp -ra mediacontrol/case/ ${ISP_BIN_DIR}
 cp -ra dewarp/dewarp_config/ ${ISP_BIN_DIR}
@@ -119,6 +127,6 @@ cp -a ${ISP_LIB_DIR}/* ${ISP_BLOB_DIR}
 
 #cd ..
 tar zcvf ${BUILD_OUTPUT_DIR}.tgz ${BUILD_OUTPUT_DIR}
-#cd 
+#cd
 
 
-- 
2.34.1

