From 0d909e1ad1961f9dcf5686104471165a4c705ad2 Mon Sep 17 00:00:00 2001
From: Kobus Goosen <kgoosen@videologyinc.com>
Date: Tue, 8 Apr 2025 13:46:42 +0200
Subject: [PATCH] cam-overlays-add-sony

---
 cam-overlays | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/cam-overlays b/cam-overlays
index 89da85d..718048f 100644
--- a/cam-overlays
+++ b/cam-overlays
@@ -1,5 +1,10 @@
+0x6a gmsl-serializer.dtbo
 0x40 lvds2mipi.dtbo
 0x3c ov5640.dtbo
 0x38 vid_isp_ar0234.dtbo
 0x36 os08a20.dtbo
-0x57 sony-mipi.dtbo
\ No newline at end of file
+0x57 sony-mipi.dtbo
+0x3d imx900.dtbo
+0x37 imx662.dtbo
+0x10 imx676.dtbo
+0x1a imx678.dtbo
--
2.34.1

